lib/util/concerns/hud_lists_2024.rb

```diff
@@ -886,11 +925,11 @@ def rental_subsidy_type(id, reverse = false)
     # 3.15.1
     def relationships_to_hoh
       {
-        1 => 'Self (head of household)',
-        2 => 'Child',
-        3 => 'Spouse or partner',
-        4 => 'Other relative',
-        5 => 'Unrelated household member',
+        1 => 'Self (Head of Household)',
+        2 => "Head of household's child",
+        3 => "Head of household's spouse or partner",
+        4 => "Head of household's other relation member",
```

Comment:
Interesting that this is the language HUD provides.  @gigxz didn't we recently override these because they didn't really make sense in the app?

Reply:
Yeah can we change these back to the old language? If you want to keep them like this in the warehouse, I can add a custom Enum for HMIS that has the labels we want.

Reply:
I agree, I like the old language better.  We should override.